site/docs/pages/index.mdx

```diff
@@ -62,27 +86,86 @@ bun add @coinbase/onchainkit
 
       </div>
       <a href="/getting-started" className="font-sans justify-self-end border rounded-xl border-solid border-white px-4 py-3 font-bold">
-        View docs 
+        View docs
       </a>
     </div>
 
   </div>
 </div>
+<div className="flex justify-between items-center md:mb-[64px] mx-auto max-w-screen-xl">
+  <div className="text-[40px] dark:text-white font-normal w-[365px] leading-[48px] tracking-[-1.6px]">
+    <p>
+      Ready-to-use onchain components
+    </p>
+  </div>
+  <div className="flex gap-12">
+    <div className="flex flex-col items-center gap-3">
+      <div className="w-[70px] h-[70px] bg-gray-800 rounded-xl p-5">
+        <svg width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We should probably have those SVGs in a different folder, so we can keep this file a bit easier to scroll.

Reply:
Also what happen when I click to the SVG? should I probably go to the doc section?

Reply:
i added the functionality so that it should link to the corresponding section below